service-print/src/main/java/org/oskari/print/PrintService.java

```diff
@@ -29,24 +26,12 @@ public PrintService(WMTSCapabilitiesCache wmtsCapsCache) {
     }
 
     public BufferedImage getPNG(PrintRequest request) throws ServiceException {
-        request.setLayers(filterLayersWithZeroOpacity(request.getLayers()));
         return PNG.getBufferedImage(request, wmtsCapsCache);
     }
 
     public void getPDF(PrintRequest request, PDDocument doc)
             throws IOException, ServiceException {
-        request.setLayers(filterLayersWithZeroOpacity(request.getLayers()));
         PDF.getPDF(request, doc, wmtsCapsCache);
     }
 
-    private static List<PrintLayer> filterLayersWithZeroOpacity(List<PrintLayer> layers) {

```

Comment:
This function shouldn't be removed. In one commit it was moved `GetPrintHandler` which was ~okay (breaks the functionality for other Action routes making straight calls to PrintService, but such routes do not exist, yet atleast), but now it is just removed.